libs/wingsdk/src/util/util.ts

```diff
@@ -0,0 +1,16 @@
+import { Duration } from "../std";
+
+/**
+ * A collection of utility functions
+ */
+export class Util {
+  /**
+   * Returns a Promise that resolves after the given Duration has passed.
+   *
+   * @param duration The duration that has to pass before the Promise is resolved.
+   * @returns a Promise that resolves after the given Duration has passed.
```

Comment:
In async functions, no need to state the obvious (that a promise is returned). And then, this function basically doesn't return anything.

```suggestion
```